eip-draft_babyjubjub.md

```diff
@@ -0,0 +1,363 @@
+---
+eip:      2494
+title:    Baby Jubjub Elliptic Curve
+authors:  Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
+discussions-to: https://ethereum-magicians.org/t/eip-2494-baby-jubjub-elliptic-curve/3968
+status:   Draft
+type:     Standards Track 
+category: ERC
+created:  2020-01-29
+---
+
+# Simple Summary
```

Comment:
Majority of EIPs, including the template, use two levels of indentation for headers:
```
## Simple Summary
```

Could you change the EIP accordingly (and also indent any subheader once more)?

Reply:
```suggestion
## Simple Summary
```